### ISSUE TITLE
CMake does not find any libraries

### ISSUE THREAD

**Issue by MarBg92** [msys2/MINGW-packages#9653]
I already installed MSYS2 from scratch.

I installed the MINGW64 toolcchain, cmake, ninja and libraries by:
`pacman -S mingw-w64-x86_64-toolchain mingw-w64-x86_64-glew mingw-w64-x86_64-sdl2 mingw-w64-x86_64-glm mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja`

I created a sample CMakeLists.txt:

```
cmake_minimum_required(VERSION 3.8)

project(Project1)

find_package(SDL2 REQUIRED)
find_package(GLEW REQUIRED)
find_package(OpenGL REQUIRED)
find_package(glm REQUIRED)

message(STATUS "SDL2 library: ${SDL2_LIBRARIES}")
message(STATUS "GLEW library: ${GLEW_LIBRARIES}")

add_executable(Project1 main.cpp)
```

Running cmake, but it tells me nothing about the libraries I asked for (it seems that it does not fill the variables):

```
$ cmake -S . -B build/
-- Building for: Ninja
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/msys64/mingw64/bin/cc.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/msys64/mingw64/bin/c++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found GLEW: C:/msys64/mingw64/lib/cmake/glew/glew-config.cmake
-- Found OpenGL: opengl32
-- SDL2 library:
-- GLEW library:
-- Configuring done
-- Generating done
-- Build files have been written to: C:/msys64/home/Mario/build
```

**Comment by MehdiChinoune** [comment]
The libraries were found. otherwise It won't succeed.
It looks like cmake is using Config files which doesn't define those variables `SDL2_LIBRARIES` and `GLEW_LIBRARIES`. but for sure they have other names that you have to search for them on the official websites of those libraries.

**Comment by MarBg92** [comment]
The same variable names work on Ubuntu 20.04.

But I have found the issue: SDL 2.0.14 (shipped with MSYS) provides only SDL2::SDL2 and SDL 2.0.10 (Ubuntu 20.04) provides only SDL2_LIBRARIES. This is fixed in Ubuntu 20.10 (which also comes with SDL 2.0.14), so it was their fault.

GLEW also wants the GLEW::GLEW syntax. Works now. Thank you.
